x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -176,6 +184,112 @@ elastic-agent inspect output --output default --program filebeat
 <hr>
 ++++
 
+[discrete]
+[[elastic-agent-install-command]]
+== elastic-agent install
+
+Install {agent} permanently on the system and manage it by using the system's
+service manager. The agent will start automatically after installation is
+complete.
+
+We recommend that you run this command as the root user because some
+integrations require root privileges to collect sensitive data. This command
+overwrites the `elastic-agent.yml` file in the agent directory.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent install [--ca-sha256 <string>] [--certificate-authorities <string>]
+                      [--enrollment-token <string>] [--force] [--help]
+                      [--insecure] [--kibana-url <string>]
+                      [global-flags]
+----
+
+[discrete]
+=== Options
+
+//REVIEWERs: Left out --staging because I think we decided previously that we
+//would not document that option becasue it's for internal use.
+
+`--ca-sha256 <string>`::
+Comma-separated list of certificate authority hash pins used for certificate
+verification.
+
+`--certificate-authorities <string>`::
+Comma-separated list of root certificates used for server verification.
+
+`--enrollment-token <string>`::
+Enrollment token generated by {fleet}. You can use the same enrollment token for
+multiple agents.
+
+`--force`::
+Force overwrite of current policy without prompting for confirmation.
+This flag is helpful when using automation software or scripted deployments.
+
+`--help`::
+Show help for the `install` command.
+
+`--insecure`::
+Allow an insecure connection to {kib}. When this flag is specified, API keys are
+sent in clear text. We strongly recommend that you use a secure connection.
+
+`--kibana-url <string>`::
+URL of the {kib} endpoint where {fleet} is running.
+
+{global-flags-link}
+
+[discrete]
+=== Examples
+
+[source,shell]
+----
+elastic-agent install -f --kibana-url=http://localhost:5601 \
+  --enrollment-token=OEV0bmauVUI0a3dmdWc1T3Bad1o6VGxCa3U4UEFTQ0NycbBSUFwoazVBdx==
```

Comment:
With this example you will need to add `--insecure` because your kibana URL is not `https://`